README.md

```diff
@@ -52,35 +58,15 @@ Then login as NOMIS_BATCHLOAD user and go to ‘manage locations’ and then 'AB
 - `npm run clean` -> cleans previously generated files
 - `npm run build` -> cleans and regenerates assets.
 - `npm run typecheck` -> runs typescript type checking
+- `npm run start:feature` -> runs the application configured to run integration tests
```

Comment:
Added a new mechanism of running the app configured for tests as it's really annoying to have to comment out .env file